webview/src/plots/components/comparisonTable/styles.module.scss

```diff
@@ -189,22 +189,48 @@ $gap: 4px;
   display: none;
 }
 
+.classButtons {
+  font-family: $font;
```

Comment:
Updated the font since it takes up less space and it felt like it made more sense since we use it for plot section widgets:

<img width="1156" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/e4cbbebd-2906-4e04-8184-36a0fc52c0fe">